apps/spruce/src/components/PatchesPage/index.tsx

```diff
@@ -117,16 +116,14 @@ export const PatchesPage: React.FC<Props> = ({
         <StatusSelector />
         {filterComp}
         <CheckboxContainer>
-          <Checkbox
-            data-cy="commit-queue-checkbox"
-            onChange={commitQueueCheckboxOnChange}
-            label={
-              pageType === "project"
-                ? "Only Show Commit Queue Patches"
-                : "Include Commit Queue"
-            }
-            checked={isCommitQueueCheckboxChecked}
-          />
+          {pageType === "project" && (
+            <Checkbox
```

Comment:
Now that this checkbox only comes from the Project Patches, I think it makes sense to pass it in with the `filterComp` prop and push up `[isGitHubMergeQueueCheckboxChecked, setIsGitHubMergeQueueCheckboxChecked]`. Then you can decouple `includeMergeQueuePatches` from the checkbox when it comes from UserPatches